packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -609,4 +600,30 @@ export class KeytipLayerBase extends BaseComponent<IKeytipLayerProps, IKeytipLay
     this.setState({ inKeytipMode: inKeytipMode });
     this._keytipManager.inKeytipMode = inKeytipMode;
   };
+
+  /**
+   * Emits a warning if duplicate keytips are found for the current keytip
+   */
+  private _detectDuplicateKeytips = (): void => {
```

Comment:
From the name of this function isn't very clear what this does, maybe something like `WarnIfDuplicateKeytipsAtLevel`

Reply:
Sure